src/processor/reconnection-graph.service.ts

```diff
@@ -151,13 +151,16 @@ export class ReconnectionGraphService {
         }
       }
     } catch (err) {
+      this.logger.error(`Error updating graph for user ${dsnpUserStr}, provider ${providerStr}: ${err}`);
       if (updateConnections) {
-        this.logger.error(`Error updating graph for user ${dsnpUserStr}, provider ${providerStr}: ${err}`);
+        /// if updateConnections is true, we want to queue a graph update job and pause the queue
         this.graphUpdateQueue.add('graphUpdate', data_nt, { jobId: jobId_nt });
-      } else {
-        this.logger.error(err);
-        throw err;
       }
+      // if we have unknwon error, capacity error we want to pause the queue
+      if (err instanceof errors.UnknownError || err instanceof errors.CapacityLowError || err instanceof errors.GetUserGraphError) {
+        this.graphUpdateQueue.pause();
```

Comment:
if capacity is low, or provider endpoint malfunction or an unknown error we dont expect (probably due to configuration issues) to pause the queue